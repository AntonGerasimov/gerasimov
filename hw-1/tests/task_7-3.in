4.0009765625
