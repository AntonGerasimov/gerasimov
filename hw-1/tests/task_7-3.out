01000000000000011000000000001000000000000000000000000000000000000000000000000000
